dts: arduino101: Fix BT Timeout error

SavinayDharmappa opened the pull request:
Generate BLUETOOTH_UART_ON_DEV_NAME and
UART_PIPE_ON_DEV_NAME configs from device tree.


Signed-off-by: Savinay Dharmappa <savinay.dharmappa@intel.com>

galak reviewed (changes requested):
We should do this via dts.  We should come up with zephyr,bt-uart, zephyr,bt-mon-uart, zephyr,uart-pipe.

I've got a patch to support zephyr,console that I'll push soon to give an example

galak commented:
Take a look at PR #641 to see how I did things for zephyr,console.

SavinayDharmappa committed:
devicetree: Generate BLUETOOTH_UART,UART_PIPE etc config from dt

patch uses chosen property zephyr,bt-uart, zephyr,uart-pipe
and zephyr,bt-mon-uart to determine the uart instance to be
used for bluetooth,uart_pipe and bluetooth_monitor and generate
appropriate configs.

Signed-off-by: Savinay Dharmappa <savinay.dharmappa@intel.com>

SavinayDharmappa committed:
dts: arduino_101: Add zephyr,bt-uart, zephyr,uart-pipe to dt

patch adds zephyr,bt-uart, zephyr,uart-pipe  and zephyr,bt-mon-uart
to device tree as chosen property.

Signed-off-by: Savinay Dharmappa <savinay.dharmappa@intel.com>

SavinayDharmappa committed:
dts: frdmk_64f: Generate UART_PIPE config from device tree

patch adds zephyr,uart-pipe to devie tree as chosen property.
This is added for frdmk64f because sanitycheck test was
breaking with changes of generating configs for UART_PIPE
from device tree instead of Kconfig.

Signed-off-by: Savinay Dharmappa <savinay.dharmappa@intel.com>

SavinayDharmappa commented:
@galak , please review this patch.
